rero_ils/modules/api.py

```diff
@@ -185,6 +186,7 @@ def create(cls, data, id_=None, delete_pid=False,
         return record
 
     @classmethod
+    @memoized(timeout=3)
```

Comment:
I'm not fan about cached a record for 3 seconds. A lot of thing can happens during this delay on an object. But if
the cached is removed/replace if the object is update/delete/reindex, it could ba a good solution but with a lot of test. 

Probably we need to implement a performance test (using JMeter ?) to benchmark this "performance issue"

Reply:
Still a draft and not ready jet.